### ISSUE TITLE
 Keyboard focus is not clearly visible inside the calendar field.

### ISSUE THREAD

**Issue by msftedad** [ant-design/antd-issue-helper#326]
Keyboard focus is not clearly visible inside the calendar filed.

**Test Environment:**
Microsoft Windows 11 Enterprise Version 10.0.22621 Build 22621

**Repro steps:**
Hit the URL: [DatePicker - Ant Design Demo (codepen.io)](https://codepen.io/dummy123123/pen/XWoRYyO)
TAB to any calendar filed and expand it.
Verify whether keyboard focus is clearly visible inside the calendar field or not.

**Actual Results:**
 Keyboard focus is not clearly visible inside the calendar field.

**Expected Results:**
 Keyboard focus should be clearly visible inside the calendar field.

**Attachments:**

https://github.com/ant-design/antd-issue-helper/assets/93735775/b39ed047-a53b-463f-90ef-a9ac4832edea

**Comment by msftedad** [comment]
Hi team, Any update on this issue?
